zarr/creation.py

```diff
@@ -21,7 +21,7 @@ def create(shape, chunks=True, dtype=None, compressor='default',
            overwrite=False, path=None, chunk_store=None, filters=None,
            cache_metadata=True, cache_attrs=True, read_only=False,
            object_codec=None, dimension_separator=None, write_empty_chunks=True,
-           *, zarr_version=None, **kwargs):
+           *, zarr_version=None, storage_transformers=(), **kwargs):
```

Comment:
All the other are None when not used, so suggest that here too

Reply:
Using a tuple instead was explicitly proposed in the comment in https://github.com/zarr-developers/zarr-python/pull/1096#pullrequestreview-1055940239, no strong opinion from my side.